### DISCUSSION TITLE
TimeColumn in GeoJson item not working ?

### DISCUSSION THREAD

**Discussion by LHKaay** [TerriaJS/terriajs#6797] [Q&A] [answered]
Hi,
have someone work with GeoJson item before?
Why the time column is not working !!

**Comment by na9da** [comment, marked as answer]
Hi @LHKode,

For GeoJSON items, you can enable time dimension by specifying the `timeProperty` to use for the dataset. Current limitation is that it only accepts a date syntax that can be parsed by the Javascript [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format) class. 

So for eg, if your geojson has a feature property called `eventdate` formatted something like `2023-06-22 10:00:00`, in your catalog item definition, you could set the `timeProperty` like below:

```
{
   "type": "geojson",
   "name": "Some geojson",
   "timeProperty": "eventdate",
   ...
}
```

I have [created a ticket](https://github.com/TerriaJS/terriajs/issues/6809) to add support for custom date formats with `timeProperty`.

**Reply by LHKaay** [reply]
Hi @na9da,
Thanks for your helpful answer. I will keep eyes on this update.
